python/lsst/qa/explorer/postprocess.py

```diff
@@ -23,358 +23,19 @@
-    return newDf
-
-
-class MultibandPostprocessConfig(PostprocessConfig):
-    filterMap = DictField(keytype=str, itemtype=str,
-                          default={'HSC-G': 'g', 'HSC-R': 'r',
-                                   'HSC-I': 'i', 'HSC-Z': 'z',
-                                   'HSC-Y': 'y'},
-                          doc="Dictionary mapping full filter name to short one " +
-                              "for column name munging.")
-    camelCase = Field(dtype=bool, default=False,
-                      doc="Write per-filter columns names with camelCase, else underscore "
-                          "For example: gPsfFlux instead of g_PsfFlux.")
-    multilevelOutput = Field(dtype=bool, default=True,
-                             doc="Whether results dataframe should have a " +
-                                 "multilevel column index (True) or be flat " +
-                                 "and name-munged (False).")
-
-
-class MultibandPostprocessTask(PostprocessTask):
-    """Do the same set of postprocessing calculations on all bands
-
-    This is identical to `PostprocessTask`, except for that it does the
-    specified functor calculations for all filters present in the
-    input `deepCoadd_obj` table.  Any specific `"filt"` keywords specified
-    by the YAML file will be superceded.
-    """
-    _DefaultName = "multibandPostprocess"
-    ConfigClass = MultibandPostprocessConfig
-
-    def run(self, parq, funcs=None, dataId=None):
-        dfDict = {}
-        for filt in parq.columnLevelNames['filter']:
-            analysis = self.getAnalysis(parq, funcs=funcs, filt=filt)
-            df = analysis.df
-            if dataId is not None:
-                for key, value in dataId.items():
-                    df[key] = value
-            dfDict[filt] = df
-
-        # This makes a multilevel column index, with filter as first level
-        df = pd.concat(dfDict, axis=1, names=['filter', 'column'])
-
-        if not self.config.multilevelOutput:
-            noDupCols = analysis.noDupCols
-            if dataId is not None:
-                noDupCols += list(dataId.keys())
-            df = flattenFilters(df, self.config.filterMap, noDupCols=noDupCols,
-                                camelCase=self.config.camelCase)
+from lsst.utils import getPackageDir
 
-        return df
+from lsst.pipe.tasks.postprocess import TransformObjectCatalogTask, TransformObjectCatalogConfig
 
 
-class WriteQATableConfig(MultibandPostprocessConfig):
+class WriteQATableConfig(TransformObjectCatalogConfig):
     def setDefaults(self):
         self.functorFile = os.path.join(getPackageDir("qa_explorer"),
                                         'data', 'QAfunctors.yaml')
 
 
-class WriteQATableTask(MultibandPostprocessTask):
+class WriteQATableTask(TransformObjectCatalogTask):
     """Compute columns of QA interest from coadd object tables
 
     Only difference from MultibandPostprocessTask is the default
```

Comment:
Doc string: "Only difference from `TransformObjectCatalogTask` is the default functorFile."